views/projects.html

```diff
@@ -0,0 +1,50 @@
+{% extends "layout.html" %}
+{% block body %}
+  <script src="{{HTTP_STATIC_URL}}friendlycode/js/require-config.js"></script>
+  <script src="{{HTTP_STATIC_URL}}friendlycode/vendor/require.min.js"></script>
+  <div id="container">
+    <h2>Your Projects</h2>
+    <table id="project-list">
+
+    {% for project in projects %}
+      <tr class="project bramble-user-project"
+          id="{{ [ 'project', project.id ] | join('-') }}"
+      {% for key, value in project %}
+        {{ [ "data", "project", key ] | join("-") }}="{{ value }}"
+      {% endfor %}
+      >
+        <td class="side-fill"></td>
+        <td class="project-title">
+          {{ project.title }}
+          <span class="project-information">
+          </span>
+        </td>
+        <td class="project-delete">
+          Delete
+        </td>
+        <td class="button-fill"></td>
+         <td class="project-remix">
+          Remix
+        </td>
+        <td class="side-fill"></td>
+      </tr>
+    {% endfor %}
+
+    </table>
+
+    <div class="project-button" id="project-0">Create New</div>
+
+    <div id="modal_wrapper">
+      <div id="modal_window">
+
+      <div style="text-align: right;"><a id="modal_close" href="#">Close <b>X</b></a></div>
+
+      <p><label>Project Name<br>
+      <input id="project-name" type="text" autofocus required size="48" name="name" value=""></label></p>
```

Comment:
This was code borrowed along with the modal javascript.  As far as I understood, it wasn't meant to be final.